k8s/cmd/commands/flags/client_flags.go

```diff
@@ -105,4 +108,10 @@ func ApplyConnectOpts(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(&DeploymentName, "deployment-name", "", "", "the name of vineyard deployment")
 	cmd.Flags().IntVarP(&Port, "port", "", 9600, "the port of vineyard deployment")
 	cmd.Flags().IntVarP(&ForwardPort, "forward-port", "", 9600, "the forward port of vineyard deployment")
+	cmd.Flags().IntVarP(&Verbose, "log-verbose", "", 0,
+		"the log level to print, default is 0, support:"+"\n"+
+			"- info(0): print info log."+"\n"+
+			"- debug(1): print debug log."+"\n"+
+			"The output will be info log by default, if you set the verbose to 1, "+
+			"the output will be debug log and info log.")
```

Comment:
`log-level`, not `log-verbose`. I believe you never see any such naming in other libraries. You can have a simple mapping like

- error -> 5
- warn -> 4
- info -> 3
- debug -> 2
- ....

And just let user to decide "info" / "debug" / "warn" / "error", don't expose the internal number to user.

Reply:
Make sense to me.